src/olympia/lib/akismet/admin.py

```diff
@@ -1,10 +1,123 @@
 from django.contrib import admin
+from django.contrib.admin.utils import flatten_fieldsets
+from django.shortcuts import get_object_or_404
+from django.utils.html import format_html
+from django.utils.translation import ugettext, ugettext_lazy as _
 
 from .models import AkismetReport
+from .tasks import submit_to_akismet
 
 
+class ReportedStatusFilter(admin.SimpleListFilter):
+    title = _('Reported Status')
+    parameter_name = 'reported_status'
+    REPORTED_HAM = 'ham'
+    REPORTED_SPAM = 'spam'
+    UNREPORTED = 'unreported'
+    QUERYSET_LOOKUP = {
```

Comment:
Let's maybe name this filters? Also, we should add a comment maybe (feel free to decide for your self, I'm undecided) that notes that the items in `VALUES` should be represented in `QUERYSET_LOOKUP` too, afaik you'll get an error otherwise.

Reply:
https://github.com/mozilla/addons-server/pull/9238/commits/daa1ecd770674af73266c4659fce233526928459